configure.ac

```diff
@@ -108,10 +117,11 @@ AC_CHECK_LIB(magic, magic_descriptor, , [AC_MSG_ERROR([libmagic not found])], -l
 AC_CHECK_LIB(cap-ng, capng_change_id, , [AC_MSG_ERROR([libcap-ng not found])], -lcap-ng)
 AC_CHECK_LIB(seccomp, seccomp_rule_add, , [AC_MSG_ERROR([libseccomp not found])], -lseccomp)
 AC_CHECK_LIB(lmdb, mdb_env_create, , [AC_MSG_ERROR([liblmdb not found])], -llmdb)
+AC_CHECK_LIB(md, MD5Final, , [AC_MSG_ERROR([libmd is missing])], -lmd)
```

Comment:
Need to check if this is really needed unconditionally now or what.

Also, we should really change a bunch of these to `PKG_CHECK_MODULES` at another point.

Reply:
Committed accidentally with the init file stuff, I _think_ it's debian-specific, bit it's on my todo to track down.